src/frontend/components/HomeSection/HomeSection.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import "./HomeSection.css";
+
+export const HomeSection = ({ banner }) => {
+  const { title, description, imageUrl, cta, ctaLink } = banner;
+
+  return (
+    <div className="banner">
```

Comment:
Make use of semantic HTML wherever possible, here it can be `main` tag and then inside `section`.